python/foglamp/services/core/api/update.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" FogLAMP package updater API support"""
+
+from aiohttp import web
+import uuid
+import asyncio
+import json
+
+# FogLAMP imports
+from foglamp.services.core import server
+from foglamp.common import logger
+
+_LOG_LEVEL_DEBUG = 10
+_logger = logger.setup(__name__, level=_LOG_LEVEL_DEBUG)
```

Comment:
before merge let's convert to default as warning?!

Reply:
yes